Sources_v3/StreamChatUI/ChatChannel/ChatFileAttachmentListView+ItemView.swift

```diff
@@ -0,0 +1,179 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+extension ChatFileAttachmentListView {
+    open class ItemView: View, UIConfigProvider {
+        public var attachment: _ChatMessageAttachment<ExtraData>? {
+            didSet { updateContentIfNeeded() }
+        }
+
+        public var tapHandler: () -> Void = {}
+
+        public var actionIconSize = CGSize(width: 20, height: 20)
+
+        // MARK: - Subviews
+
+        public private(set) lazy var fileIconImageView: UIImageView = {
+            let imageView = UIImageView()
+            imageView.contentMode = .scaleAspectFit
+            return imageView.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var fileNameLabel: UILabel = {
+            let label = UILabel()
+            label.font = UIFont.preferredFont(forTextStyle: .body).bold
+            label.adjustsFontForContentSizeCategory = true
+            return label.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var fileSizeLabel: UILabel = {
+            let label = UILabel()
+            label.font = UIFont.preferredFont(forTextStyle: .footnote)
+            label.adjustsFontForContentSizeCategory = true
+            return label.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var activityIndicator: UIActivityIndicatorView = {
+            let indicator = UIActivityIndicatorView()
+            indicator.hidesWhenStopped = true
+            return indicator.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var actionIconImageView: UIImageView = {
+            let imageView = UIImageView()
+            imageView.contentMode = .scaleAspectFit
```

Comment:
If user provide image of different size than we expects what should happen:
1. image resize for constant size (right now)
2. layout updates for image size